docs/docs-ref-autogen/outlook/office.enhancedlocation.yml

```diff
@@ -87,6 +103,26 @@ items:
 
 
       `getAsync(callback?: (result: Office.AsyncResult<LocationDetails[]>) => void): void;`
+
+      #### Examples
+
+
+      ```javascript
+
+      Office.context.mailbox.item.enhancedLocation.getAsync(callbackFunction);
+
+
+      function callbackFunction(asyncResult) {
+          asyncResult.value.forEach(function (place) {
+                  console.log("Display name: " + place.displayName);
+                  console.log("Type: " + place.locationIdentifier.type);
+                  if (place.locationIdentifier.type == Office.MailboxEnums.LocationType.Room) {
+                      console.log("Email address: " + place.emailAddress);
+              }
+          });
+      }
```

Comment:
Need to fix the indentation in this code sample (in the snippet file where this code sample originates from) -- the first several lines within the `forEach` are indented 4 spaces too far.